.markdownlint.yml

```diff
@@ -0,0 +1,2 @@
+# disable 'Trailing punctuation in heading' rule, as we want to use ? at the end of titles
+MD026: false
```

Comment:
Are we using `markdownlint` in this project? I can't find any references to it

Reply:
I added to my Code
Interested to see what it talks about ...